### PR TITLE
fix(Suite/Test): untitled suite/test-case #1632

### PR DESCRIPTION
**Pull request by a8m**
This PR replace #1634.

@mochajs/mocha can we merge it to **v3.0.0**?

### CODE REVIEW

**Comment by dasilvacontin** [comment]
LGTM but for the comment I added.

**Comment by dasilvacontin** [comment]
ping @a8m

**Comment by dasilvacontin** [comment]
Closes https://github.com/mochajs/mocha/issues/1632.

**Comment by a8m** [comment]
@dasilvacontin can you PTAL? thx

File: test/suite.js [hunk 1]
```diff
@@ -393,4 +393,42 @@ describe('Suite', function(){
...
+
+  describe('initialization', function() {
+    it('should throw an error if the title isn\'t a string', function() {
+      (function() {
+        new Suite(undefined, 'root');
+      }).should.throw();
+
+      (function() {
+        new Suite(function(){}, 'root');
+      }).should.throw();
```
Comment (dasilvacontin) [inline test/suite.js]: Could these assert the error message?

Comment (dasilvacontin) [inline test/suite.js]: And with these I mean every added `.should.throw` in the PR.

**Comment by dasilvacontin** [comment]
One new comment, sorry I didn't notice ealier. :/

Thanks for picking the PR up!

**Comment by boneskull** [comment]
merged via 442b6a0e3d7ce929f6990f0d3c3d48e936f664a2 and dc6ca3f165010dde37aea69c82651af8df20ac7f in v3.0.0 branch

**Comment by a8m** [comment]
:clap:

**Pull request closed without merging**